internal/pod_models_test.go

```diff
@@ -940,6 +963,76 @@ var _ = Describe("pod_models", func() {
 			})
 		})
 
+		Context("with a different process group crash looping in main container", func() {
+			BeforeEach(func() {
+				cluster.Spec.Buggify.CrashLoopContainers = []fdbv1beta2.CrashLoopContainerObject{
+					{
+						ContainerName: fdbv1beta2.MainContainerName,
+						Targets:       []string{"storage-2"},
+					},
+				}
+				spec, err = GetPodSpec(cluster, fdbv1beta2.ProcessClassStorage, 1)
+				Expect(err).NotTo(HaveOccurred())
+			})
+
+			It("should have the normal start command", func() {
+				mainContainer := spec.Containers[0]
+				Expect(mainContainer.Name).To(Equal(fdbv1beta2.MainContainerName))
+				Expect(mainContainer.Args).To(Equal([]string{
+					"fdbmonitor --conffile /var/dynamic-conf/fdbmonitor.conf" +
+						" --lockfile /var/dynamic-conf/fdbmonitor.lockfile" +
+						" --loggroup operator-test-1" +
+						" >> /var/log/fdb-trace-logs/fdbmonitor-$(date '+%Y-%m-%d').log 2>&1",
+				}))
+			})
+		})
+
+		Context("with a different process group crash looping in container", func() {
+			BeforeEach(func() {
+				cluster.Spec.Buggify.CrashLoopContainers = []fdbv1beta2.CrashLoopContainerObject{
+					{
+						ContainerName: fdbv1beta2.MainContainerName,
+						Targets:       []string{"storage-1"},
+					},
+				}
+				spec, err = GetPodSpec(cluster, fdbv1beta2.ProcessClassStorage, 1)
+				Expect(err).NotTo(HaveOccurred())
+			})
+
+			It("should have the normal start command", func() {
+				mainContainer := spec.Containers[0]
+				Expect(mainContainer.Name).To(Equal(fdbv1beta2.MainContainerName))
+				Expect(mainContainer.Args).To(Equal([]string{"crash-loop"}))
+
+				sidecarContainer := spec.Containers[1]
+				Expect(sidecarContainer.Name).To(Equal(fdbv1beta2.SidecarContainerName))
+				Expect(sidecarContainer.Args).ToNot(Equal([]string{"crash-loop"}))
+			})
+		})
+
+		Context("with a different process group crash looping in container", func() {
+			BeforeEach(func() {
+				cluster.Spec.Buggify.CrashLoopContainers = []fdbv1beta2.CrashLoopContainerObject{
+					{
+						ContainerName: fdbv1beta2.SidecarContainerName,
+						Targets:       []string{"storage-1"},
+					},
+				}
+				spec, err = GetPodSpec(cluster, fdbv1beta2.ProcessClassStorage, 1)
+				Expect(err).NotTo(HaveOccurred())
+			})
+
+			It("should have the normal start command", func() {
```

Comment:
```suggestion
			It("should set the correct start commands", func() {
```

Separating the It command into two statements would make the failure more obvious e.g. if the issue is in the sidecar setup or the main container setup, but I'm fine with leaving it as it is.